index.html

```diff
@@ -152,11 +154,19 @@ <h6 class="client">Canopy</h6>
                     <li class="list-item"><button type="button" class= "btn btn-tags"> javaScript</button></li>
                 </ul>
                 <div class="action">
-                    <button  type="button" class="btn normal-b"> See project</button>
+                    <button  type="button" class="btn normal-b" data-modal-target="#popup3"> See project</button>
                 </div>
             </div>
         </div>
-        <div class="about-myself" id="about">
+
+        <div class="cards-container" id="card-container-one">
```

Comment:
- [ ] Your pop-windows design looks great 👍🏽 but the design can be improved if you increase the height of the popup and also the height of the image to make it nicer. Also, you can increase the spacing (Padding/Marging) At the top of your modal 👍